repository/l.json

```diff
@@ -1904,6 +1904,16 @@
 				}
 			]
 		},
+		{
+			"name": "Lumi",
+			"details": "https://github.com/smadeja/lumi",
```

Comment:
Please add a labels key with "color scheme", to make it easier to understand what kind of package this is and make it easier to find.

Reply:
Absolutely, it's labelled now